x-pack/metricbeat/module/statsd/server/server.go

```diff
@@ -165,28 +165,26 @@ func buildMappings(config []StatsdMapping) (map[string]StatsdMapping, error) {
 
 func (m *MetricSet) getEvents() []*mb.Event {
```

Comment:
If there's a change in the logic of an existing function, it would be good to add a small comment for the change in the function. This will serve as a good reference to the change as well as a simple documentation.